build_teamcity

```diff
@@ -42,7 +42,9 @@ rm -rf wheelhouse/
 _scope_closed "cleanup"
 
 _scope_opened "setup"
-# Force Python stdout/err to be unbuffered.
+
+# Force Python stdout/err to be unbuffered to
+# have immediate feedback in TeamCity build logs.
 export PYTHONUNBUFFERED="notemtpy"
```

Comment:
OK - looks like this got copied to a few places:
```
./build_dcos_launch.sh:export PYTHONUNBUFFERED="notemtpy"
./build_local.sh:export PYTHONUNBUFFERED="notemtpy"
./build_teamcity:export PYTHONUNBUFFERED="notemtpy"
./gen/build_deploy/bash/installer_internal_wrapper.in:export PYTHONUNBUFFERED="notemtpy"
```

Feel free to ignore as it is completely unrelated to the current PR, I just spotted it and I'm happy to send a separate PR to fix this.

Reply:
hehe